boruta/boruta/readme.md

```diff
@@ -0,0 +1,70 @@
+# Boruta algorithm
```

Comment:
we could just say here `see [../1-click-boruta/readme.md](blah blah)` no? or even do a `ln -s`... is there any difference between the two?